CLANc/CLANc.munki.recipe

```diff
@@ -84,6 +88,27 @@
 				<dict>
 					<key>version</key>
 					<string>%version%</string>
+					<key>installs</key>
+					<array>
+						<dict>
+							<key>CFBundleIdentifier</key>
+							<string>%NAME%</string>
+							<key>CFBundleName</key>
+							<string>%NAME%</string>
+							<key>CFBundleShortVersionString</key>
+							<string>%version%</string>
+							<key>CFBundleVersion</key>
+							<string>For the Version number, use "About Clan..." inside the application</string>
+							<key>md5checksum</key>
+							<string>%md5checksum%</string>
+							<key>path</key>
+							<string>/Applications/%NAME%/%NAME%.app</string>
+							<key>type</key>
+							<string>application</string>
+							<key>version_comparison_key</key>
+							<string>md5checksum</string>
```

Comment:
This will lead to munki downgrading newer versions of the app to the version currently in deployment. Any chance we could use `CFBundleShortVersionString` instead?

Reply:
My understanding was that the original problem was the inconsistency in the in the versioning of the `CFBundleShortVersionString` that is why I used the `md5checksum` to compare. But from what I have seen the `CFBundleShortVersionString` looked consistent so I think it would be alright to use that.